src/main/java/org/example/Employee.java

```diff
@@ -0,0 +1,45 @@
+package org.example;
+
+public class Employee {
+    private int emplyoeeId;
```

Comment:
You spelled employee wrong here, which might lead to confusion later! Other than that, looks good!

Reply:
Good catch; fixed!